packages/zoom-in/src/components/zoom.js

```diff
@@ -129,6 +119,14 @@ const Zoom = props => {
 
     zoomedRef.current.addEventListener('transitionend', handleOpenEnd)
     setZoom(true)
+    overlayRef.current.cursor = 'pointer'
+    overlayRef.current.cursor = 'zoom-out'
+    zoomedRef.current.setAttribute(
+      'style',
+      `cursor: pointer;
+      cursor: zoom-out;`
```

Comment:
Is `cursor: pointer` a typo?

Reply:
`cursor: pointer` is a fallback for IE which doesn't support `cursor: zoom-out`.